packages/pds/tests/seeds/basic.ts

```diff
@@ -38,6 +38,7 @@ export default async (sc: SeedClient) => {
     [img1, img2], // Contains both images and a quote
     sc.posts[bob][0].ref,
   )
+  await sc.post(carol, posts.carol[0])
```

Comment:
I see `posts.carol[0]` used above—is this intended to be the newly-added post `posts.carol[1]`?

Reply:
goddammit yes 😅